api/src/test/java/uk/gov/dhsc/htbhf/claimant/ClaimantServiceIntegrationTestsWithScheduledServices.java

```diff
@@ -123,6 +123,7 @@ void shouldSendWeWillLetYouKnowEmailForEligibleApplicantWhoseEmailOrPhoneDontMat
         assertThat(payment.getPaymentAmountInPence()).isEqualTo(expectedEntitlement.getTotalVoucherValueInPence());
 
         assertThatEmailWithNameOnlyWasSent(claim, PENDING_DECISION);
+        assertThatLetterWithAddressOnlyWasSent(claim, LetterType.INSTANT_SUCCESS_CHILDREN_MATCH);
```

Comment:
We'll need another test for the alternative letter

Reply:
This is covered in the parameterized unit test. I'd argue that we don't need both options at integration level.

Reply:
Given this affects the external integration (sending a letter), I'd argue it is worthwhile

Reply:
Made the test parameterized.